js_modules/dagit/packages/core/src/instance/InstanceBackfills.tsx

```diff
@@ -84,23 +84,7 @@ export const InstanceBackfills = () => {
             <div>
               {isBackfillHealthy ? null : (
                 <Box padding={{horizontal: 24, vertical: 16}}>
-                  <Alert
-                    intent="warning"
-                    title="The backfill daemon is not running."
-                    description={
-                      <div>
-                        See the{' '}
-                        <a
-                          href="https://docs.dagster.io/deployment/dagster-daemon"
-                          target="_blank"
-                          rel="noreferrer"
-                        >
-                          dagster-daemon documentation
-                        </a>{' '}
-                        for more information on how to deploy the dagster-daemon process.
-                      </div>
-                    }
-                  />
+                  <DaemonNotRunningAlertBody />
```

Comment:
I believe this page can / should use the `<DaemonNotRunningAlert />` and the new fragment,  but it's doing some other things and appears (maybe?) to support multiple backfill daemons? Will talk to some folks and see. For now, just re-using the body at least cleans things up a bit!